src/screens/Proposals.js

```diff
@@ -265,6 +265,9 @@ const IdAndTitle = ({ id, name, selectProposal }) => {
       <span
         css={`
           color: ${theme.surfaceContentSecondary};
+          white-space:normal;
+          float:left;
+          text-align:left;
```

Comment:
What do you think if we apply this styles to the parent `Link` component? In that case we can drop the `float` prop.

Reply:
Hi , yes sorry i'll try to get it figured out. Sorry still new to using github 😅

Reply:
haha no to worry!  Thank you

Reply:
Haha alright. I'm still learning as i'm going, as far as I can see you recommended me to apply the style to the parent Link component. I've however been having a hard time finding this as I am new to the React infrastructure. How I see it is that the modified Link component is part of `const IdAndTitle` , which is in turn part of a large DataView `Proposals.js line 185-235 ` I've been trying a couple of things out applying style changes for different components to no success.  I feel like I'm not looking at the right place, tips would be very appreciated as all of this is a learning process for me :)

Reply:
Ah sorry, maybe i should have been more specific! I was referring to do something like this:

```
    <Link
      onClick={handleOnClick}
      css={`
        white-space: normal;
        text-align: left;
      `}
    >
      <span
        css={`
          color: ${theme.surfaceContentSecondary};
        `}
      >
        {name}
      </span>
    </Link>
```

Reply:
Done! Sorry for the wait. Also, by taking this styling approaches the progress bar is automatically fixed aswell, so for that nothing else had to be pushed. To recreate: On Desktop mode when resizing the screen resolution of the browser, the progress bar will wrap properly.